.github/workflows/bench_datasheet.yml

```diff
@@ -55,6 +55,7 @@ jobs:
 
     env:
       FEATURE: ${{ matrix.feature }}
+      SCCACHE_IDLE_TIMEOUT: 0
```

Comment:
Why don't we add this to `.github/actions/sccache` so we don't need to duplicate it everywhere

Reply:
I suggested that would not apply the env var globally, but only for the duration of the run of the composite. I might have been wrong though.

Reply:
Env vars can be 'exported' from the composite via: https://github.com/risc0/risc0/blob/main/.github/actions/sccache/action.yml#L45

Reply:
This was added. Build times good, checks all passed.